docs/integrate.md

```diff
@@ -1,49 +1,70 @@
 
 #### Using HTML
 
-If you build your application via HTML/PWA, Ultra will launch the webpage inside android’s web-view. You can pick your existing mobile website and reuse it for Ultra after making minor changes. 
+Ultra launches the webpage inside a mobile’s web-view. Here, you may pick your existing mobile website and mould it for Ultra after few tweaks. 
+
+> This approach helps you in building a decent mobile application but it does not appear as realistic and responsive as the one developed using React Native.
+
+!!!note One important point to keep in mind is that both the approaches do not allow you to navigate away from your main domain. Let us suppose your application is hosted on some domain for example, `ABC.com` and the site contains a hyperlink that navigates control to some other domain say, `ABC.org`. In such case, the site does not work and throws a `Security Error` message. If you have a similar situation in your application, please contact [us](contact.md) as we have got a way for bypassing this limitation by whitelisting the domains you own.
+
+## 2: Integrate Ultra JavaScript SDK
+After you have made up your mind on which UI platform to work with (either React Native or HTML/PWA), include Ultra’s JavaScript SDK into your application. This SDK helps you build applications that run within Flipkart app and give access to the Ultra’s JavaScript methods required for logins, authorizations (OAuth) and payments.
+
+!!!note The logic presented here works in both React Native and HTML/PWA. These methods are asynchronous and always return a [promise](https://javascript.info/promise-basics) that gets resolved with some values except few other JavaScript methods that have no return value.
+
+Next, [add the dependency](clients.md#step-1) for the SDK. [Contact us](contact.md) for generating the parameters specific to your application i.e. clientID and secret to access the Ultra APIs.
+
+After adding the dependency, [initialize the SDK](clients.md#step-2) using the above parameters.
+
+## 3: Ultra User Login Process
+In this section, you learn how to fetch user details securely from Flipkart’s domain. A user can gain entry into your application only after allowing permissions to share their information (such as name, email, phone number etc.). This step guides you about the generation of grant token, fetching user’s information and login the user into your application.
+
+### Generate grant token
+You might have come across the terms which are very common in the technical world of authorization (OAuth) world such as “Granted Scopes” and “Grant tokens”. To know more, read [this](https://alexbilbie.com/guide-to-oauth-2-grants/).
```

Comment:
The word "world" is used twice in the this part of the sentence:
 "... in the technical **world** of authorization (OAuth) **world** such as ..."